Products/PleiadesEntity/skins/PleiadesEntity/location_edit.pt

```diff
@@ -87,8 +87,12 @@
     </table>
 
 <script tal:attributes="src string:${context/portal_url}/location_map.js?v=20251201-1"></script>
+<script tal:define="token python:context.portal_registry.get('pleiades.mapbox.access_token', '')"
```

Comment:
I'm not sure we can safely use `context.portal_registry.get('pleiades.mapbox.access_token', '')` in templates when we need to get this value (it may work here, but probably won't work in skin templates as in pleiades.notredame).

I would suggest instead providing a helper view or simple JSON API to get the value, so that we're not calling protected API from the template.

Additionally, I would suggest adding the `window.PLEIADES_MAPBOX_TOKEN` value in `main_template.pt` (in pleiades.notredame) so that the value is available everywhere. That should simplify both of these PRs.